packages/atri-app-core/src/canvasMachine.ts

```diff
@@ -193,6 +194,22 @@ function setMousePosition(
   context.mousePosition = event.event;
 }
 
+function setProbableParent(
+  context: CanvasMachineContext,
+  event: MOUSE_MOVE_EVENT
+) {
+  const { target } = event.event;
+  if (target !== null && "closest" in target) {
+    const parentEl = (target as HTMLElement).closest("[data-atri-parent]");
+    let parentElId: string | null = null;
+    if (parentEl) {
+      parentElId = parentEl.getAttribute("data-atri-comp-id")!;
+    }
+    if (parentElId !== null && parentElId !== context.probableParent)
+      context.probableParent = parentElId;
+  }
```

Comment:
Emit two events, `DROPZONE_DESTROYED` and `DROPZONE_CREATED`, if the old probable parent and the new probable parent are not the same.

Add an extra condition for `DROPZONE_DESTROYED` - that condition is the old parent must not be null.

Reply:
Done